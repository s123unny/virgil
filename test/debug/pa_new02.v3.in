b ../core/pa_new02.v3 21
run
bt
info l
c
q
