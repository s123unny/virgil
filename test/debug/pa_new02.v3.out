breakpoint|../core/pa_new02.v3 21|0
stop|stopOnBreakpoint
bt|pa_new02.main|../core/pa_new02.v3|21
variable|0|this|pa_new02|pa_new02|false
variable|1|a|0|int|false
variable|2|b|0|int|false
variable|3|c|0|int|false
stop|end
result|0
